mappings/net/minecraft/client/gui/screen/SplashScreen.mapping

```diff
@@ -1,11 +1,11 @@
 CLASS net/minecraft/class_425 net/minecraft/client/gui/screen/SplashScreen
-	FIELD field_17767 reloadMonitor Lnet/minecraft/class_4011;
+	FIELD field_17767 reload Lnet/minecraft/class_4011;
 	FIELD field_17770 progress F
-	FIELD field_17771 applyCompleteTime J
+	FIELD field_17771 reloadCompleteTime J
 	FIELD field_18217 client Lnet/minecraft/class_310;
 	FIELD field_18218 exceptionHandler Ljava/util/function/Consumer;
 	FIELD field_18219 reloading Z
-	FIELD field_18220 prepareCompleteTime J
+	FIELD field_18220 reloadStartTime J
```

Comment:
They updated reloads in snapshots, so old names are likely correct and new names wrong